specification/description.yml

```
@@ -251,6 +258,17 @@ introduction: |
       . . .
   ```
 
+    ### Sample Rate Limit Headers When Burst Limit is Reached:
+
+  ```
+      . . .
+      ratelimit-limit: 1200
```

```suggestion
      ratelimit-limit: 5000
```
Let's use a more realistic value.